c-deps/libroach/ccl/key.cc

```diff
@@ -0,0 +1,23 @@
+// Copyright 2017 The Cockroach Authors.
+//
+// Licensed as a CockroachDB Enterprise file under the Cockroach Community
+// License (the "License"); you may not use this file except in compliance with
+// the License. You may obtain a copy of the License at
+//
+//     https://github.com/cockroachdb/cockroach/blob/master/licenses/CCL.txt
+
+#include "key.h"
+#include <filters.h>  // CryptoPP
```

Comment:
I'm putting comments here because it's not otherwise clear what these are for. cryptopp doesn't have an `include` subdir. Any ideas? I don't particularly want to re-organize everything.